### PR TITLE
Updating code-sample for v0.27.0

### PR DESCRIPTION
**Pull request by alallema**
Following [this issue](https://github.com/meilisearch/integration-guides/issues/190)
Updating one and addition of new samples.

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -300,6 +329,34 @@ settings_guide_sortable_1: |-
...
+    'minWordSizeForTypos' => [
+      'twoTypos' => 12
+    ],
+    'disableOnAttributes' => [
+      'title'
+    ]
+  ]);
+typo_tolerance_guide_1: |-
+  $client->index('movies')->updateTypoTolerance([
+    'enabled' => False
```
Comment (brunoocasali) [inline .code-samples.meilisearch.yaml:343]:
```suggestion
    'enabled' => false
```

**Review by brunoocasali — CHANGES_REQUESTED** [review]
Good job! 🥇 🎉 

I found some small items I think they could be addressed before the merge :)

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -121,6 +121,13 @@ update_settings_1: |-
     'synonyms' => [
       'wolverine' => ['xmen', 'logan'],
       'logan' => ['wolverine']
+    ],
+    'typoTolerance' => [
+        'minWordSizeForTypos' => [
+            'oneTypo' => 8,
+            'twoTypos' => 10
+        ],
+        'disableOnAttributes' => ['title']
```
Comment (bidoubiwa) [inline .code-samples.meilisearch.yaml:130]: I think indentation should be 2 spaces

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Pull request merged**